pkg/virt-handler/vm.go

```diff
@@ -700,11 +700,22 @@ func (d *VirtualMachineController) migrationTargetUpdateVMIStatus(vmi *v1.Virtua
 	}
 
 	// Handle post migration
-	if domainExists && vmi.Status.MigrationState != nil && !vmi.Status.MigrationState.TargetNodeDomainDetected {
+	if domainExists &&
+		domain.Status.Status == api.Running &&
```

Comment:
Previously `TargetNodeDomainDetected` had the information of whether the domain was detected (running or not). Now it will be true only if it is running. Is it intentional? There could be any side effects from this change?
As is, it seems that the two information are redundant, since we can drop the boolean in favor of the new timestamp.
Maybe I'm being too punctilious, but how about separate the two things? 
Thanks

Reply:
There are side effects. this impacts the ability to live migrate paused VMs. I'll fix this.

Reply:
I fixed the paused VM case.

The `TargetNodeDomainDetected` boolean has been used to indicate to the source node that the target node has received the active qemu process. This was being performed inaccurately in the case of post-copy because the target node receives the domain before the migration has completed.

The change in behavior here is that we don't say the target has received the domain until we detect the domain is either `Running` or in a user initiated Paused state.